web/src/components/tasks/business-structure/BusinessStructureTask.tsx

```diff
@@ -68,6 +68,7 @@ export const BusinessStructureTask = (props: Props): ReactElement => {
 
     queueUpdateTaskProgress(props.task.id, "COMPLETED");
     await updateQueue.queueProfileData(profileData).update();
+    await updateQueue.queueFormationFormData(business.formationData.formationFormData).update();
```

Comment:
Why is this change necessary?

Reply:
Because the address no longer persists when the user interacts with the business structure task.